nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -118,50 +125,49 @@ def _apply(self,
 
         nncf_graph = NNCFGraphFactory.create(model)
         node_and_bias_value = (
-            (node, self._backend_entity.get_bias_value(node, nncf_graph, model)) \
-                for node in nncf_graph.get_all_nodes() if self._backend_entity.is_node_with_bias(node, nncf_graph)
+            (node, self._backend_entity.get_bias_value(node, nncf_graph, model))
+            for node in nncf_graph.get_all_nodes()
```

Comment:
Such formatting looks weird.

> for node in nncf_graph.get_all_nodes()

Looks like missed `:`, not as part of the unnamed expression.